iceoryx_utils/test/moduletests/test_cxx_vector.cpp

```diff
@@ -17,6 +17,8 @@
 #include "iceoryx_utils/cxx/vector.hpp"
 #include "test.hpp"
 
+#include <vector>
```

Comment:
Please update the copyright header so that there is a separate line for each copyright owner.